src/components/Dashboard/TodoList.vue

```diff
@@ -364,7 +364,7 @@
                         <v-icon>mdi-menu-down</v-icon>
                       </v-btn>
                     </template>
-                    <v-list class="actions__more-actions">
+                    <v-list class="actions__more-actions pb-2 pt-2">
```

Comment:
This can be simplified to `py-2`.

Ref: https://vuetifyjs.com/en/styles/spacing/